changelog/min_runtime_classes.dd

```diff
@@ -0,0 +1,100 @@
+Interfaces and classes can be used without the runtime if only static fields are utilized
+
+Prior to this release any attempt to use interfaces and/or classes without the runtime would result
+in compile-time errors due to the lack of `TypeInfo`.  However, as long as classes are not instantiated
+there is no need for `TypeInfo`.
+
+Beginning with this release the compiler will no longer emit errors about missing `TypeInfo` when
+using interfaces and/or classes as long as they are not instantiated and only `shared static` members are
+utilized.
+
+$(B Example 1)
+---
+module object
+
+private alias extern(C) int function(char[][] args) MainFunc;
+private extern (C) int _d_run_main(int argc, char** argv, MainFunc mainFunc)
+{
+    return mainFunc(null);  // assumes `void main()` for simplicity
+}
+---
+
+---
+interface I
+{
+    shared static int i;
+}
+
+class A : I
+{
+    shared static int a;
+}
+
+class B : A
+{
+    shared static int b;
+}
+
+void main()
+{
+    B.b = B.a + B.i;
+}
+---
+
+$(CONSOLE
+dmd -conf= -defaultlib= main.d object.d -of=main
+size main
+   text    data     bss     dec     hex filename
+   1867    1208      32    3107     c23 main
+)
+
+Non-shared static members can also be used, but will require a thread-local storage (TLS) implementation.
+For Linux, the TLS implementation is already supplied by the C runtime and C
```

Comment:
Don't we know that there are only two platforms where it doesn't work out of the box?

Reply:
All the other tests in the auto-tester passed except osx32 and Win32_64.  So those are the only supported platforms I know of that don't work out of the box.

Reply:
Exactly, so why only mentioning `Linux`?

Reply:
I'm just using Linux and an illustrative example.  How else should I say it?  "For *nix platforms..."?

Reply:
I'll add "For example.."